blivet/tasks/availability.py

```diff
@@ -399,6 +403,7 @@ def available_resource(name):
 BLOCKDEV_BTRFS_PLUGIN = blockdev_plugin("btrfs", BLOCKDEV_BTRFS_TECH)
 BLOCKDEV_CRYPTO_PLUGIN = blockdev_plugin("crypto", BLOCKDEV_CRYPTO_TECH)
 BLOCKDEV_DM_PLUGIN = blockdev_plugin("dm", BLOCKDEV_DM_TECH)
+BLOCKDEV_DM_PLUGIN_RAID = blockdev_plugin("dm", BLOCKDEV_DM_TECH_RAID)
```

Comment:
It would be a bit clearer if you named it `BLOCKDEV_DM_RAID_PLUGIN` instead.

Reply:
But I guess I understand if you don't want to imply that there is a separate plugin for dmraid, since there isn't.

Reply:
It's just a part of the dm plugin, so I think it makes more sense to name it `BLOCKDEV_DM_PLUGIN_RAID`